Remotion/Web/Core/ExecutionEngine/WxeResourcePageStep.cs

```diff
@@ -34,6 +34,9 @@ public class WxeResourcePageStep : WxePageStep
     /// <summary>
     ///   Calls the page using the calling assemby's resource directory.
     /// </summary>
+#if !NETFRAMEWORK
+    [Obsolete("Use a constructor with type/assembly argument instead.", error: true, DiagnosticId = "RMWEB0002")]
```

Comment:
@a-ctor should we start maintaining a type with the const DiagnosticIDs so we don't acccidentally reuse?